smdebug/core/tfevent/util.py

```diff
@@ -8,6 +8,12 @@
 from .proto.tensor_pb2 import TensorProto
 from .proto.tensor_shape_pb2 import TensorShapeProto
 
+try:
+    import tensorflow.compat.v1 as tf
+except ImportError:
+    tf = None
```

Comment:
Having imports silently be None seems likely to cause bugs down the road.

Reply:
I can add a log here. we need this exception handler so that the code doesn't fail on non-TF containers

Reply:
If there's a TF-specific method, maybe it should be refactored out of `smdebug/core`